iquip/apps/stage.py

```diff
@@ -8,21 +8,25 @@
 
 from sipyco.pc_rpc import Client
 from PyQt5.QtCore import QObject, Qt, pyqtSignal, pyqtSlot
+from PyQt5.QtWidgets import (
+    QAbstractSpinBox, QDoubleSpinBox, QPushButton, QWidget,
+    QVBoxLayout, QHBoxLayout,
```

Comment:
Isn't it better to order alphabetically?
Indeed, I don't know orderling like this is the best..

Reply:
Okay, I will rearrange them!